client/src/partials/cash/cash.html

```diff
@@ -50,7 +50,7 @@
                 <bh-currency-select
                   currency-id="CashCtrl.payment.currency_id"
                   validation-trigger="CashVoucherForm.$submitted"
-                  on-change="CashCtrl.digestExchangeRate()"
+                  on-change="CashCtrl.digestExchangeRate(currency)"
                   disable-ids="CashCtrl.disabledCurrencyIds"
                   cashbox-id="CashCtrl.cashbox.id"
```

Comment:
Why a component responsible of changing currency must be some how directly linked to the cash box?